locator/network_topology_strategy.cc

```diff
@@ -338,7 +338,8 @@ future<tablet_map> network_topology_strategy::reallocate_tablets(schema_ptr s, t
     load_sketch load(tm);
     co_await load.populate(std::nullopt, s->id());
 
-    tablet_logger.debug("Allocating tablets for {}.{} ({}): dc_rep_factor={} tablet_count={}", s->ks_name(), s->cf_name(), s->id(), _dc_rep_factor, tablets.tablet_count());
+    tablet_logger.debug("Allocating tablets for {}.{} ({}): dc_rep_factor={} tablet_count={}",
+                        s->ks_name(), s->cf_name(), s->id(), _dc_rep_factor, tablets.tablet_count());
```

Comment:
nit: cosmetic change, right?
can be eliminated (does your editor do this automatically?)